srcpkgs/colord-kde/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'colord-kde'
+pkgname=colord-kde
+version=0.5.0
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="kcoreaddons extra-cmake-modules gettext pkg-config qt5-devel qt5-qmake"
+makedepends="kcmutils-devel libXrandr-devel lcms2-devel plasma-framework-devel"
+depends="colord $(vopt_if gcm gnome-color-manager)"
+short_desc="KDE colord integration"
+maintainer="Danila Kryukov <danila.kryukov.dev@yandex.ru>"
+license="GPL-2.0-or-later"
+homepage="https://cgit.kde.org/colord-kde.git/"
+distfiles="${KDE_SITE}/colord-kde/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=6c545009af737ad67900e4d8549086bc65b979c799b2402871bba44addb52d2f
+
+build_options="gcm"
+desc_option_gcm="Enable support for gnome callibration tools"
```

Comment:
If this is just for runtime dependency, there is no need for build option. Leave it for user to find out what's needed, or _maybe_ install note with vdoc.